### PR TITLE
[BIOMAGE-1566] Replaced header in settings too

### PR DESCRIPTION
**Pull request by kafkasl**

### CODE REVIEW

File: src/pages/settings/profile/index.jsx [hunk 1]
```diff
@@ -82,19 +92,21 @@ const ProfileSettings = () => {
...
-            <Col style={{ width: '40%' }}>
+        <Space direction='vertical' style={{ width: '100%', padding: '20px', background: ' white' }}>
+          <Row type='flex'>
+            <Col xl={{ span: 12, offset: 6 }} span={24}>
 
-              <Form layout='horizontal'>
-                <h2>Profile settings:</h2>
+              <Form
+                layout='horizontal'
+                {...formItemLayout}
```
Comment (aerlaut) [inline src/pages/settings/profile/index.jsx]: The linter is showing an error, can you add the required ignore for this line?

**Comment by aerlaut** [comment]
I manage to make the submit and cancel button to stick to the right side of the page :

```            
                .....  
              </Form>
            </Col>
          </Row>
          <Row>
            <Col xl={{ span: 12, offset: 6 }} span={24}>
              <Row justify='end'>
                <Space>
                  <Button
                    onClick={() => router.back()}
                  >
                    Cancel
                  </Button>
                  <Button
                    type='primary'
                    onClick={() => updateDetails()}
                  >
                    Save changes
                  </Button>
                </Space>
              </Row>
            </Col>
          </Row>
```

image

File: src/pages/settings/profile/index.jsx (hunk 1 above)
Comment (kafkasl) [reply src/pages/settings/profile/index.jsx]: I just added the properties without the variable

**Comment by kafkasl** [comment]
> I manage to make the submit and cancel button to stick to the right side of the page :
> 
> ```
>                 .....  
>               </Form>
>             </Col>
>           </Row>
>           <Row>
>             <Col xl={{ span: 12, offset: 6 }} span={24}>
>               <Row justify='end'>
>                 <Space>
>                   <Button
>                     onClick={() => router.back()}
>                   >
>                     Cancel
>                   </Button>
>                   <Button
>                     type='primary'
>                     onClick={() => updateDetails()}
>                   >
>                     Save changes
>                   </Button>
>                 </Space>
>               </Row>
>             </Col>
>           </Row>
> ```
> 
> image

awesome, done

**Review by aerlaut — APPROVED** [review]
LGTM

**Pull request merged**